pkg/products/threescale/reconciler.go

```diff
@@ -1391,7 +1391,7 @@ func (r *Reconciler) reconcileOutgoingEmailAddress(ctx context.Context, serverCl
 	} else {
 		accessToken, err := r.GetAdminToken(ctx, serverClient)
 		if err != nil {
-			r.log.Error("Failed to get admin token in reconcileOutgoingEmailAddresss", err)
+			r.log.Info("Failed to get admin token in reconcileOutgoingEmailAddresss: " + err.Error())
```

Comment:
Why was this changed to `r.log.Info` and not `r.log.Warning`? I have no preferences just asking the question. And this goes for the other instances in this file also.

Reply:
I've been following this idea ->  https://github.com/integr8ly/integreatly-operator/blob/master/pkg/products/threescale/reconciler.go#L1120-L1123
`INFO` for the cases when it is expected and nothing happened. 
`WARNING` for the cases when something went wrong (missing the value)  but I will be handled (defaulting to values) or when reconciling and got an error but the stage is not `StageFailed` yet